ena/ERC000030.json

```diff
@@ -0,0 +1,503 @@
+          }
+        },
+        "marineSampleSampleStatus": {
+          "description": "Refers to the completness of sample metadata validation. Example: preliminary but can be used to provide data discovery services.",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "marineSampleLastUpdateDate": {
+          "description": "Date of the last update of this sample in the SAMPLES REGISTRY. Example: 2014-03-01Z.",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string",
+                "format": "date"
+              }
+            }
+          }
+        },
+        "nonSampleTermsProjectName": {
+          "description": "Name of the project within which the sequencing was organized.",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "collectionEventInformationEnvironmentalPackage": {
+          "description": "MIGS/MIMS/MIMARKS extension for reporting of measurements and observations obtained from one or more of the environments where the sample was obtained. All environmental packages listed here are further defined in separate subtables. By giving the name of the environmental package, a selection of fields can be made from the subtables and can be reported",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string",
+                "enum": [
+                  "Air",
+                  "Built-environment",
+                  "Host-associated",
+                  "Human-associated",
+                  "Human-gut",
+                  "Human-oral",
+                  "Human-skin",
+                  "Human-vaginal",
+                  "Microbial mat/biofilm",
+                  "Miscellaneous natural or artificial environment",
+                  "Plant-associated",
+                  "Sediment",
+                  "Soil",
+                  "Wastewater/sludge",
+                  "Water"
+                ]
+              }
+            }
+          }
+        },
+        "collectionEventInformationEnvironmentBiome": {
+          "description": "Biomes are defined based on factors such as plant structures, leaf types, plant spacing, and other factors like climate. Biome should be treated as the descriptor of the broad ecological context of a sample. Examples include: desert, taiga, deciduous woodland, or coral reef. EnvO (v 2013-06-14) terms can be found via the link: www.environmentontology.org/Browse-EnvO",
+          "type": "array",
+          "minItems": 1,
```

Comment:
Same as before: The `minItems ` here is redundant because it's already being define in `"patternProperties"` , it can be removed from here.

Reply:
This is true for all `attributes`.